Core/CoreTests/Extensions/InstIconExtensionsTests.swift

```diff
@@ -459,60 +463,6 @@ class InstIconExtensionTests: XCTestCase {
-        XCTAssertEqual(Image.checkbox, Image("checkbox", bundle: .core))
-        XCTAssertEqual(Image.checkboxSelected, Image("checkboxSelected", bundle: .core))
-        XCTAssertEqual(Image.chevronDown, Image("chevronDown", bundle: .core))
-        XCTAssertEqual(Image.collaborations, Image("collaborations", bundle: .core))
-        XCTAssertEqual(Image.conferences, Image("conferences", bundle: .core))
-        XCTAssertEqual(Image.coursesTab, Image("coursesTab", bundle: .core))
-        XCTAssertEqual(Image.coursesTabActive, Image("coursesTabActive", bundle: .core))
-        XCTAssertEqual(Image.dashboardLayoutGrid, Image("dashboardLayoutGrid", bundle: .core))
-        XCTAssertEqual(Image.dashboardLayoutList, Image("dashboardLayoutList", bundle: .core))
-        XCTAssertEqual(Image.dashboardTab, Image("dashboardTab", bundle: .core))
-        XCTAssertEqual(Image.dashboardTabActive, Image("dashboardTabActive", bundle: .core))
-        XCTAssertEqual(Image.dropdown, Image("dropdown", bundle: .core))
-        XCTAssertEqual(Image.filterCheckbox, Image("filterCheckbox", bundle: .core))
-        XCTAssertEqual(Image.grab, Image("grab", bundle: .core))
-        XCTAssertEqual(Image.homeroomTab, Image("homeroomTab", bundle: .core))
-        XCTAssertEqual(Image.homeroomTabActive, Image("homeroomTabActive", bundle: .core))
-        XCTAssertEqual(Image.inboxTab, Image("inboxTab", bundle: .core))
-        XCTAssertEqual(Image.inboxTabActive, Image("inboxTabActive", bundle: .core))
-        XCTAssertEqual(Image.k5dueToday, Image("k5dueToday", bundle: .core))
-        XCTAssertEqual(Image.k5grades, Image("k5grades", bundle: .core))
-        XCTAssertEqual(Image.k5homeroom, Image("k5homeroom", bundle: .core))
-        XCTAssertEqual(Image.k5importantDates, Image("k5importantDates", bundle: .core))
-        XCTAssertEqual(Image.k5resources, Image("k5resources", bundle: .core))
-        XCTAssertEqual(Image.k5schedule, Image("k5schedule", bundle: .core))
-        XCTAssertEqual(Image.logout, Image("logout", bundle: .core))
-        XCTAssertEqual(Image.markReadLine, Image("markReadLine", bundle: .core))
-        XCTAssertEqual(Image.masteryLTI, Image("masteryLTI", bundle: .core))
-        XCTAssertEqual(Image.offlineLine, Image("offlineLine", bundle: .core))
-        XCTAssertEqual(Image.partialSolid, Image("partialSolid", bundle: .core))
-        XCTAssertEqual(Image.radioButtonSelected, Image("radioButtonSelected", bundle: .core))
-        XCTAssertEqual(Image.radioButtonUnselected, Image("radioButtonUnselected", bundle: .core))
-        XCTAssertEqual(Image.qrCode, Image("qrCode", bundle: .core))
-        XCTAssertEqual(Image.share, Image("share", bundle: .core))
-        XCTAssertEqual(Image.todoTab, Image("todoTab", bundle: .core))
-        XCTAssertEqual(Image.todoTabActive, Image("todoTabActive", bundle: .core))
-        XCTAssertEqual(Image.unionLine, Image("unionLine", bundle: .core))
```

Comment:
These were manually added in a previous PR so we should either revert of the removal of these or move these to a separate file to avoid being removed every time we run the icon import script.